File: week 1 C/mario.c
#include <cs50.h>
#include <stdio.h>

void make_pyramid(int n);

int main(void)
{
    int a;
    do
    {

        a = get_int("Height: ");
    }
    while (!(a <= 8 && a >= 1));

    printf("\n");
    make_pyramid(a);
}

void make_pyramid(int n)
{
    for (int i = 1; i <= n; i++)
    {
        int cnt = n - i;

        for (int j = 1; j <= n; j++)
        {
            if (cnt < j)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }

        printf("  ");

        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: Week 6 Python/week6 problem set/sentimental-mario-more/mario.py
import cs50 as np

h = np.get_int("Height: ")

while h <= 0 or h > 8:
    h = np.get_int("Height: ")


for i in range(1, h + 1, 1):
    temp = (" " * (h - i)) + ("#" * i) + ("  ") + ("#" * i)
    print(temp)
